app.js

```diff
@@ -38,30 +38,56 @@ app.get('/process_get', function (req, res) {
   // Prepare output in JSON format
   var response = {
     url: req.query.url,
+    //linkurl : req.query.linkurl,
+    //scripturl : req.query.scripturl,
   };
   // hitting url
   request(req.query.url, function (error, response, body) {
     if (!error && response.statusCode == 200) {
       // console.log(body) // Print the google web page.
       var $ = cheerio.load(response.body);
-  $('link').each(function(index,value){
+
+      function newUrl(actualurl,replaceurl){
+      var parsed = url(actualurl);
+      var newurl = parsed.set('hostname',replaceurl);
+      return newurl.href;
+    }
+
+      function replacement(target,attribute,replaceurl){
+      $(target).each(function(index,value){
+        var linkw = $(value).attr(attribute);
+        var data = newUrl(linkw,replaceurl);
+        $(this).attr(attribute,data);
+        //console.log(util.inspect(value,false,1));
+      });
+    }
+
+    
+
+    replacement('link','href',req.query.linkurl);
```

Comment:
with this approach user need to pass URL this looks not a good approach, please check it again.